File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/internal/data/framework/api/apifile/entity/FileResponse.java
package ru.usedesk.chat_sdk.internal.data.framework.api.apifile.entity;

import com.google.gson.annotations.SerializedName;

public class FileResponse {
    Integer status;
    @SerializedName("file_link")
    String fileLink;
    String size;
    String id;
    String type;
    String name;
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/data/repository/api/entity/FileResponse.kt
package ru.usedesk.chat_sdk.data.repository.api.entity

import ru.usedesk.common_sdk.api.entity.UsedeskApiError

internal class FileResponse(
    var status: Int?,
    var fileLink: String?,
    var size: String?,
    var id: String?,
    var type: String?,
    var name: String?
) : UsedeskApiError()